server/shadow/Mac/mac_shadow.c

```diff
@@ -660,7 +660,7 @@ macShadowSubsystem* mac_shadow_subsystem_new()
 	return subsystem;
 }
 
-int Mac_ShadowSubsystemEntry(RDP_SHADOW_ENTRY_POINTS* pEntryPoints)
+FREERDP_API int Mac_ShadowSubsystemEntry(RDP_SHADOW_ENTRY_POINTS* pEntryPoints)
```

Comment:
We might not want to export Win_ShadowSubsystemEntry/ X11_ShadowSubsystemEntry /Mac_ShadowSubsystemEntry. Can we still leave them internal currently?